src/cc/frontends/p4/compiler/ebpfAction.py

```diff
@@ -0,0 +1,327 @@
+from p4_hlir.hlir import p4_action, p4_field, p4_signature_ref, p4_header_instance
+import ebpfProgram
+from programSerializer import ProgramSerializer
+from compilationException import *
+import ebpfScalarType
+import ebpfCounter
+import ebpfType
+import ebpfInstance
+
+
+class EbpfActionData:
```

Comment:
All classes should inherit from object, as per guidelines starting from python 2.5.2: https://docs.python.org/release/2.5.2/ref/node33.html. This will lead to better compatibility between python2 and python3.